src/NServiceBus.AcceptanceTests/DelayedDelivery/When_deferring_a_message_to_the_past.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace NServiceBus.AcceptanceTests.DelayedDelivery
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.Features;
+    using NUnit.Framework;
+
+    public class When_deferring_a_message_to_the_past : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_deliver_message_now()
```

Comment:
:-1: I still think this may cause conflicts with the immediate dispatch feature.